vuu-ui/sample-apps/feature-filter-table/src/useFilterTable.tsx

```diff
@@ -59,6 +78,84 @@ export const useFilterTable = ({ tableSchema }: FilterTableFeatureProps) => {
     [dataSource]
   );
 
+  const hasFilterWithName = (filters: NamedFilter[], name: string) =>
+    filters.findIndex((f) => f.name === name) !== -1;
+
```

Comment:
Should we move this out of the hook as a helper function, since it doesn't really depend on any props or state?